vars/Amazon.yml

```
@@ -0,0 +1,12 @@
+---
+# vars file for Amazon Linux
+
+# The pip package names
+#
+# Amazon Linux 2 is woefully behind the times and requires Python 2 to
+# support its antiquated version of the yum package manager.
+package_names:
+  - python2-pip
+  - python-devel
```

I would rather keep these sorted than deviate with a more logical ordering just to be consistent with how we handle this stuff.

```suggestion
  - python-devel
  - python2-pip
```